doc/install/docker.md

```diff
@@ -0,0 +1,23 @@
+## Configuring Docker for Elasticsearch
+For correctly deploying Elasticsearch, configuration of Docker must be tuned. Please refer to [Elasticsearch's official guide][1].
```

Comment:
use directly the url?

Reply:
URL is in the footnote because it's used twice